packages/app/src/specs/HighlightedText.vue

```diff
@@ -21,8 +24,8 @@
 import { computed } from 'vue'
 
 const props = withDefaults(
-  defineProps<{text: string, indexes: number[], highlightClasses?: string}>(),
-  { text: '', indexes: () => [], highlightClasses: 'text-white' },
+  defineProps<{text: string, indexes: number[], highlightClasses?: string, titleText?: string, truncate?: boolean}>(),
```

Comment:
Seems truncate will never be false from what I can see? Do we need this prop?

Reply:
There are some cases where `truncate` will be false:

- Use in `SpecItem.vue`. In this case, the outer div has the `truncate` class on it since there are two instances of `HighlightedText` being used side by side in this component (one for the file name and one for the extension), so using the `truncate` prop wouldn't do what we want in this case.
- Use in `SpecFileItem.vue`. Similar situation
- Use in `DirectoryItem.vue`. In this case the truncation is handled by the parent component as well.

I am re-thinking adding this as a prop though... seems like we should handle the truncation outside of the `HighlightedText` component. I'll mess with that